documentation/docs/guides/openapi.md

```diff
@@ -63,6 +63,52 @@ func helloWorld(c fuego.ContextNoBody) (string, error) {
 }
 ```
 
+## Group Options, Options Groups & Custom Options
+
+You can also customize the OpenAPI specification for a group of routes.
+
+```go
+package main
+
+import (
+	"github.com/go-fuego/fuego"
+	"github.com/go-fuego/fuego/option"
+)
+
+// Define a reusable group of options
+var optionPagination = option.Group(
+	option.QueryInt("page", "Page number", param.Default(1)),
+	option.QueryInt("limit", "Items per page", param.Default(10)),
+)
+
+// Custom options for the group
+var customOption = func(r *fuego.BaseRoute) {
+	r.XXX  = YYY // Direct access to the route struct to inject custom behaviour
```

Comment:
I guess the spellcheck tool was written by an American 😄

Reply:
lol, fixed